.github/workflows/auto-merge-branches.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow is intended to keep certain branches in sync and will automatically create PR's between 
+# these branches. For now these PR's must be manually reviewed and merged. Consider if we should just 
+# automatically do this if there are no conflicts. 
+#
+# This step will fail if a PR already exists, so we ignore failures. This also 
+# mean that if some other error occur, the error will also be swallowed. This is acceptable.
+name: Auto merge change between branches
+
+on:
+  push:
+    branches:
+      # List of "from" branches. Targets are defined by the `find-target-branch`-step below.
+      - releases
+      - master
+
+jobs:
+  main:
+    name: Create PR with changes that should be merged into another branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v3
+
+      - name: Set target branch
+        id: find-target-branch
+        shell: sh
+        run: |
+          if [ "${GITHUB_REF#refs/heads/}" = "master" ]; then echo '::set-output name=branch::releases/ktor2-support'; fi
```

Comment:
I assume that it keeps
- `release/ktor2-support` up to date with `master`, and 
- `master` up to date with `releases` 
But maybe add a comment in the file